src/ray/gcs/gcs_client/test/gcs_client_reconnection_test.cc

```diff
@@ -27,9 +30,9 @@
 #include "ray/rpc/gcs_server/gcs_rpc_client.h"
 #include "ray/util/util.h"
 
-using namespace std::chrono_literals;
-using namespace ray;
-using namespace std::chrono;
+using namespace std::chrono_literals;  // NOLINT
```

Comment:
No change needed, it's totally fine now: next time it would be nice to include the reason besides the `NOLINT`.